src/components/AccordionPanel/AccordionPanelContent.tsx

```diff
@@ -33,43 +28,30 @@ export const AccordionPanelContent: FC<Props> = ({ children, className = '' }) =
     node.style.height = 'auto'
   }
 
-  const handleExit = useCallback(
-    (node: HTMLElement) => {
-      updateNodeHeight(node, wrapperRef)
-    },
-    [wrapperRef],
-  )
-
-  const handleExiting = useCallback(
-    (node: HTMLElement) => {
-      updateNodeHeight(node, wrapperRef)
-    },
-    [wrapperRef],
-  )
-
   const handleExited = (node: HTMLElement) => {
     node.style.height = '0px'
   }
 
   return (
     <Transition
       in={isInclude}
-      onEntering={handleEntering}
+      onEntering={togglePanel}
       onEntered={handleEntered}
-      onExit={handleExit}
-      onExiting={handleExiting}
+      onExit={togglePanel}
+      onExiting={togglePanel}
       onExited={handleExited}
       timeout={{
-        enter: 300,
+        enter: 200,
         exit: 0,
       }}
     >
       {(status) => (
         <CollapseContainer
           id={`${name}-content`}
+          role="region"
           className={`${status} ${className}`}
           aria-labelledby={`${name}-trigger`}
-          aria-hidden={!isInclude}
+          hidden={!isInclude}
```

Comment:
`hidden` 属性は意味があるので、ここは `aria-hidden` が良さそう。

> The hidden attribute must not be used to hide content that could legitimately be shown in another presentation. For example, it is incorrect to use hidden to hide panels in a tabbed dialog, because the tabbed interface is merely a kind of overflow presentation — one could equally well just show all the form controls in one big page with a scrollbar.

https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute

`hidden` はCSS無効環境で非表示になるので、HTMLのみになったときに見せたいか見せたくないかで考えると良さそうです。

Reply:
`hidden` にしないと裏側でフォーカスする気がするんですが、それは `focusable=false` で良いのかしら。

Reply:
あー隠し方によりそうです。`display: none` とか `visibility: hidden` ならフォーカスはあたらないハズ。

Reply:
`visibility: hidden` で対応した